pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -139,6 +139,22 @@ func toApiReadOnly(src bool) *ReadOnly {
 	return nil
 }
 
+// Convert_v1_Agent_To_api_Channel creates the channel for guest agent communication
+func Convert_v1_Agent_To_api_Channel(source *v1.GuestAgent, channel *Channel) error {
+
+	channel.Type = "unix"
+	channel.Source = ChannelSource{
+		Mode: "bind",
+		Path: "/var/lib/libvirt/qemu/f16x86_64.agent",
```

Comment:
@petrkotas Can you please make the value a const variable you are using it in many places in the code so if for some reason we need to change it I will be easy.